### PR TITLE
Fixed the Dune 2000 legacy map import

### PR DESCRIPTION
**Pull request by Mailaender**
Fixes https://github.com/OpenRA/OpenRA/issues/10387 and also adds a lint test so it won't happen again. Spotted another one with it: `palacec`.

### CODE REVIEW

File: OpenRA.Mods.D2k/Lint/CheckImportActors.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+namespace OpenRA.Mods.D2k.Lint
+{
+	public class CheckImportActors : ILintRulesPass
+	{
+		public void Run(Action<string> emitError, Action<string> emitWarning, Ruleset rules)
+		{
+			foreach (var actorData in D2kMapImporter.ActorDataByActorCode.Values)
+			{
+				if (!rules.Actors.Keys.Contains(actorData.First))
```
Comment (RoosterDragon) [inline OpenRA.Mods.D2k/Lint/CheckImportActors.cs]: `Keys.Contains` -> `ContainsKey`

**Comment by RoosterDragon** [comment]
:+1:

File: OpenRA.Mods.D2k/Lint/CheckImportActors.cs (hunk 1 above)
Comment (Mailaender) [inline OpenRA.Mods.D2k/Lint/CheckImportActors.cs]: I assume this is faster?

Comment (RoosterDragon) [inline OpenRA.Mods.D2k/Lint/CheckImportActors.cs]: Yes - the first is doing the lookup using the dictionary, the second requires a linear search.

**Comment by penev92** [comment]
Thanks :+1:

**Comment by penev92** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/b4384aa5d000b87650fb1bdec2b14bc64a987ad1)

**Pull request merged**